amundsen_application/static/js/components/DashboardPage/QueryList/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
```

Comment:
Cover this file with at least one test, mostly to encourage testing of future work if this component becomes more complicated.